src/definitions.ts

```diff
@@ -84,4 +84,10 @@ export interface RecordExceptionOptions {
    * Only available for iOS.
    */
   domain?: string;
+  /**
+   * A stacktrace generated by stacktrace.js
+   *
+   * Cannot be combined with `code` and `domain`
+   */
+  stacktrace?: any[];
```

Comment:
Would it take much effort to define a type that is not `any` and compatible with stacktrace.js? I have no problem with stacktrace.js and I like to mention it. However, it would be nice to know which object is expected here.

Reply:
Sure, this totally makes sense. We also only use 3 attributes of the stacktrace `stacktrace-js` provides. Thanks for the hint!